cookies/operations.py

```diff
@@ -172,3 +172,32 @@ def merge_resources(resources, master_id=None, delete=True, user=None):
     if delete:
         to_merge.delete()
     return master
+
+def add_resources_to_collection(resources, collection):
+    """
+    Adds selected resources to a collection.
+
+    Number of resources should be greater than or equal to 1.
+    And one collection has to be selected
+
+    Returns the collection after making changes.
```

Comment:
Great docstring! For methods like this, it can be helpful to write an explicit table of parameters and return values. When we generate API documentation later, some parsers will recognize the content and generate pretty HTML tables. For example (below your existing text):

```
Parameters
-------------
resources : ``QuerySet``
    The :class:`.Resource` instances that will be added to ``collection``.
collection : :class:`.Collection`
    The :class:`.Collection` instance to which ``resources`` will be added.

Returns
---------
collection : :class:`.Collection`
    Updated :class:`.Collection` instance.
```

The odd-looking references are [reST](http://www.sphinx-doc.org/en/stable/rest.html) -- I try to use this wherever possible. It's not strictly necessary, but does make documentation easier.